horizon_bsn/api/rest/neutron.py

```diff
@@ -0,0 +1,261 @@
+#
+#    (c) Copyright 2015 Hewlett-Packard Development Company, L.P.
```

Comment:
nit: either use a standard apache header or a header with bigswitch name. you're attributing your work to HP 😛